includes/Checker/Checks/Plugin_Readme_Check.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Checker\Checks\Plugin_Readme_Check
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Checker\Checks;
+
+use WordPress\Plugin_Check\Checker\Check_Result;
+
+/**
+ * Check the plugins readme.txt file and contents.
+ *
+ * @since n.e.x.t
+ */
+class Plugin_Readme_Check extends Abstract_File_Check {
+
+	/**
+	 * Check the readme.txt file.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param Check_Result $result The Check Result to amend.
+	 * @param array        $files  Array of plugin files.
+	 */
+	protected function check_files( Check_Result $result, array $files ) {
+		// Find the readme file.
+		$readme = self::filter_files_by_regex( $files, '/readme\.txt/' );
```

Comment:
Would be good to include `$`, otherwise this may match folders that are named like that (uncommon but possible).
```suggestion
		$readme = self::filter_files_by_regex( $files, '/readme\.txt$/' );
```

Reply:
Thanks @felixarntz this has been updated.